public/static/docs/command-reference/diff.md

```diff
@@ -46,148 +57,142 @@ for example when `dvc init` was used with the `--no-scm` option.
 ```dvc
-$ dvc diff HEAD^
-dvc diff from df613bc to ed10968
-
-diff for 'data/data.xml'
-+data/data.xml with md5 a304afb96060aad90176268345e10355
-
-added file with size 37.9 MB
+$ dvc diff
 ```
 
-## Example: Specific targets across Git commits
+> It produces an empty result if there are no changes to any files or
+> directories in the current workspace.
 
-We can base this example in the [Metrics](/doc/get-started/metrics) and
-[Compare Experiments](/doc/get-started/compare-experiments) chapters of our _Get
-Started_ section, that describe different experiments to produce the `model.pkl`
-file. Our example repository has the `bigrams-experiment` and
-`baseline-experiment`
-[tags](https://github.com/iterative/example-get-started/tags) respectively to
-reference these experiments.
+## Example: Comparing workspace with arbitrary commits
 
 <details>
 
-### Click and expand to setup example
+### Click and expand to setup the example
 
-Having followed the previous example's setup, move into the
-`example-get-started/` directory. Then make sure that you have the latest code
-and data with the following commands.
+Let's checkout the
+[3-add-file](https://github.com/iterative/example-get-started/releases/tag/3-add-file)
+tag, corresponding to the [Add Files](/doc/get-started/add-files) _Get Started_
+chapter, right after we added `data.xml` file with DVC:
 
 ```dvc
-$ git checkout master
-$ dvc fetch -T
+$ git checkout 3-add-file
+$ dvc pull
 ```
 
-The `-T` flag passed to `dvc fetch` makes sure we have all the data files
-related to all existing tags in the repo. You take a look at the
-[available tags](https://github.com/iterative/example-get-started/tags) of our
-example repo.
-
 </details>
 
-To see the difference in `model.pkl` among these tags, we can run the following
-command.
+To see the difference between the very previous commit of the project and the
+workspace, we can use `HEAD^` as `a_ref`:
 
 ```dvc
-$ dvc diff -t model.pkl baseline-experiment bigrams-experiment
-dvc diff from bc1722d to 8c1169d
+$ dvc diff HEAD^
+Added:
+    data/data.xml
 
-diff for 'model.pkl'
--model.pkl with md5 a664896
-+model.pkl with md5 3863d0e
-    ...
+files summary: 1 added, 0 deleted, 0 modified
 ```
 
-The output from this command confirms that there's a difference in the
-`model.pkl` file between the 2 Git commits (tags `baseline-experiment` and
-`bigrams-experiment`) we indicated.
+## Example: Comparing tags or branches
+
+<details>
 
-### What about directories?
+### Click and expand to setup the example
 
-Unlike Git, DVC features controlling entire directories without having to add
-each individual file. See `dvc add` without `--recursive` for example. `dvc run`
-can track entire directories (when these are specified as command dependencies
-or <abbr>outputs</abbr>).
+Our example repository has the `bigrams-experiment` and `baseline-experiment`
+[tags](https://github.com/iterative/example-get-started/tags) respectively to
+reference these experiments.
```

Comment:
Maybe mention `baseline-experiment` first above? And:

`...tags, that reference two different modeling experiments.`